src/preamble.js

```diff
@@ -487,27 +487,37 @@ async function getWasmBinary(binaryFile) {
 
 #if SPLIT_MODULE
 {{{ makeModuleReceiveWithVar('loadSplitModule', undefined, 'instantiateSync') }}}
-var splitModuleProxyHandler = {
+var wasmImportsProxyHandler = {
   get(target, prop, receiver) {
-    return (...args) => {
+    if (prop.startsWith('placeholder')) {
+      const moduleName = prop;
```

Comment:
Maybe simpler to just give the outer property and inner property parameters different names.
```suggestion
get(target, moduleName, receiver) {
    if (moduleName.startsWith('placeholder')) {
```

Reply:
So I thought... `placeholder` or `env` or `a` (in case minification) were kind of "environments" as opposed to "modules" that mean wasm modules. But yeah, in many places we call this 'module' as well, so.. I will change this back to `moduleName`.